app/src/main/java/com/wmods/wppenhacer/xposed/utils/MimeTypeUtils.java

```diff
@@ -0,0 +1,14 @@
+package com.wmods.wppenhacer.xposed.utils;
+
+import android.webkit.MimeTypeMap;
+
+public class MimeTypeUtils {
+    public static String getMimeTypeFromExtension(String url) {
+        String type = null;
```

Comment:
Possibility of null type if there is no extension. Better to put an empty string.